### PR TITLE
libnetworkit 6.1 (new formula)

### PR DESCRIPTION
**Pull request by fabratu**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

In the current state, the package does not support in-source builds. This is why a build-dir is created during install.

### CODE REVIEW

File: Formula/libnetworkit.rb [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+  depends_on "cmake" => :build
+  depends_on "libomp"
+  depends_on "tlx"
+
+  def install
+    mkdir "build" do
+      system "cmake", ".", *std_cmake_args,
+                           "-DNETWORKIT_EXT_TLX=#{Formula["tlx"].opt_prefix}",
+                           "-DOpenMP_CXX_FLAGS='-Xpreprocessor -fopenmp -I/usr/local/opt/libomp/include'",
```
Comment (SMillerDev) [inline Formula/libnetworkit.rb]:
```suggestion
                           "-DOpenMP_CXX_FLAGS='-Xpreprocessor -fopenmp -I#{Formula["libomp"].opt_prefix}/include'",
```

File: Formula/libnetworkit.rb [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+                           "-DOpenMP_CXX_LIB_NAMES='omp'",
+                           "-DOpenMP_omp_LIBRARY=/usr/local/opt/libomp/lib/libomp.dylib",
```
Comment (SMillerDev) [inline Formula/libnetworkit.rb]:
```suggestion
                           "-DOpenMP_omp_LIBRARY=#{Formula["libomp"].opt_prefix}/lib/libomp.dylib",
```

Comment (chenrui333) [reply Formula/libnetworkit.rb]: 👍

File: Formula/libnetworkit.rb (hunk 1 above)
Comment (chenrui333) [reply Formula/libnetworkit.rb]: 👍

**Comment by fabratu** [comment]
Thx for the updates, I will look into it.

**Comment by fabratu** [comment]
Seems the error is not related to the formula, but to the original package. The build fails due to a seq-fault in clang with the most recent version 11.0.3. This does not happen for the previous version 11.0.0 (and before). 

I can recreate the build error locally on mac OS 10.15 with clang 11.0.3 and already filled a bug report for Apple Dev. However I find confusing reports, whether or not the seq fault is plainly a bug in the compiler or pointing to something in the code.

The question is how to proceed here? Are there other packages with the similar problems?

**Pull request closed without merging**